pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitInitialScanUseCase.java

```diff
@@ -33,11 +33,6 @@ private Optional<ScanError> updateKitRequest(String hruid, String kit) {
                 return Optional.ofNullable(new ScanError(kit,
                         "Too many active kits found for \"" + hruid + "\".\n" + UserErrorMessages.IF_QUESTIONS_CONTACT_DEVELOPER));
             }
-
-            if (kitList.size() == 0) {
-                return Optional.ofNullable(new ScanError(kit, "No kit for participant with ShortId \"" + hruid + "\" was not found.\n"
-                        + UserErrorMessages.IF_QUESTIONS_CONTACT_DEVELOPER));
-            }
```

Comment:
On line 31 ^ we have already checked this so this condition was always false

Reply:
If throwing this error is helpful, May be this should go into the else of if on line# 31 
if we dont want to alter the behavior now, thats fine too.